src/DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp

```diff
@@ -32,13 +34,15 @@ struct DetAndInverseImpl;
 template <typename Symm, typename Index0, typename Index1>
 struct DetAndInverseImpl<Symm, Index0, Index1, Requires<Index0::dim == 1>> {
   template <typename T>
-  static determinant_inverse_pair<T, Symm, Index0, Index1> apply(
```

Comment:
If I correctly read through this, you've removed all uses of `determinant_inverse_pair`, so you could also delete the `using` statement on (new) line 26

Reply:
Thank you! I completely missed that...